samples/msgext-search-sso-config/nodejs/README.md

```diff
@@ -98,8 +98,12 @@ Under the root of this sample folder, build and run by commands:
 - **Interacting with the Message Extension in Teams
     Once the Messaging Extension is installed, find the icon for **Config Auth Search** in the Compose Box's Messaging Extension menu. Right click to choose **Settings** and view the Config page. Click the icon to display the search window, type anything it will show your profile picture.
 
-**Note**: If you are facing any issue in your app, please uncomment [this](https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/main/samples/msgext-search-sso-config/nodejs/index.js#L43) line and put your debugger for local debug.
-    
+**Note**: If you are facing any issue in your app, please uncomment [this](https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/main/samples/msgext-search-sso-config/nodejs/index.js#L57) line and put your debugger for local debug.
```

Comment:
Wrong line, it should be 43

Reply:
43 is old line I have updated the index code file, so the current line is 57.
![image](https://github.com/OfficeDev/Microsoft-Teams-Samples/assets/117716554/d6aeba22-dc53-41d0-b3c4-29b229ec7efa)